### ISSUE TITLE
S603 false positive on safe cmd

### ISSUE THREAD

**Issue by osauldmy** [astral-sh/ruff#17112]
### Summary

I have cmd as list of constant strings, not example with `input()` as S603 doc suggest to verify, but it still flags as untrusted.

```python
import subprocess

git_diff = subprocess.run(["/usr/bin/git", "diff", "main", "--name-status"], check=True, capture_output=True)
```

```bash
$ ruff check --select S ~/ruff_bug.py
ruff_bug.py:3:12: S603 `subprocess` call: check for execution of untrusted input
```

### Version

0.11.2

**Comment by MichaReiser** [comment]
Ignoring `run` calls where all arguments are string literals seems reasonable

**Comment by trag1c** [comment]
I'd like to have a go at this!

**Comment by Avasam** [comment]
Hi.
https://github.com/astral-sh/ruff/pull/17136 doesn't work on tuples
![image](https://github.com/user-attachments/assets/aa447131-b564-4ef0-bb6f-a2979bda3c13)
